### PR TITLE
Scoped api keys - bug fixes

### PR DESCRIPTION
**Pull request by skofman1**
There are 3 fixes in this PR (did them as one PR to avoid merge conflicts):
1. SetApiKey command should have source as part of it:

image

2. Copying a key provides no indication of success: added a popup that disappears after 1 second/
image

3. Should remove code related to 'expire api key' option

### CODE REVIEW

File: src/NuGetGallery/Views/Users/ApiKeys.cshtml [hunk 1]
```diff
@@ -188,7 +188,7 @@
                                                                 <ul class="longlist">
                                                                 @for (int i = 0; i < subjectsList.Count; i++)
                                                                 {
-                                                                    var lastInBatch = i != 0 && i%itemsPerBatch == (itemsPerBatch - 1);
+                                                                    var lastInBatch = i != 0 && i%itemsPerBatch == (itemsPerBatch - 1);
```
Comment (scottbommarito) [inline src/NuGetGallery/Views/Users/ApiKeys.cshtml]: can you remove the extra space?

File: src/NuGetGallery/Views/Users/ApiKeys.cshtml [hunk 2]
```diff
@@ -652,26 +646,33 @@
     }
 
 </script>
-<script> // Disable copy API key button after 10 mins of inactivity
+<script> // Copy API key button related
```
Comment (scottbommarito) [inline src/NuGetGallery/Views/Users/ApiKeys.cshtml]: I think this just can be `// Copy API key button`
(the `related` is implied)

File: src/NuGetGallery/Views/Users/ApiKeys.cshtml [hunk 3]
```diff
@@ -652,26 +646,33 @@
...
-                $(this)
-                    .keypress(function() {
-                        idleTime = 0;
-                    });
-            }});
+    function startCopyApiKeyButtonTimer() {
+        // Increment the idle time counter every minute.
+        idleInterval = setInterval(timerIncrement, 60000); // 1 minute
+
+        // Zero the idle timer on mouse movement.
```
Comment (scottbommarito) [inline src/NuGetGallery/Views/Users/ApiKeys.cshtml]: `// Zero the idle timer on mouse movement or key press.`

File: src/NuGetGallery/Views/Users/ApiKeys.cshtml [hunk 4]
```diff
@@ -690,8 +700,11 @@
 
     function applyActionButtonHandlers() {
         $('.remove-api-key').on('click', removeApiKey);
-        $('.expire-api-key').on('click', expireApiKey);
         $('.regenerate-api-key').on('click', regenerateApiKey);
+        
+        // Copy api key related
```
Comment (scottbommarito) [inline src/NuGetGallery/Views/Users/ApiKeys.cshtml]: I don't think this comment adds much.

File: src/NuGetGallery/Views/Users/ApiKeys.cshtml [hunk 5]
```diff
@@ -680,6 +681,15 @@
             clearInterval(idleInterval);
         }
     }
+
+    function oncopyapikeybuttonclick(e) {
```
Comment (chenriksson) [inline src/NuGetGallery/Views/Users/ApiKeys.cshtml]: nit: onCopyApiKeyButtonClick (casing)?

File: src/NuGetGallery/Views/Users/ApiKeys.cshtml [hunk 6]
```diff
@@ -690,8 +700,11 @@
 
     function applyActionButtonHandlers() {
         $('.remove-api-key').on('click', removeApiKey);
-        $('.expire-api-key').on('click', expireApiKey);
         $('.regenerate-api-key').on('click', regenerateApiKey);
+        
+        // Copy api key related
+        $('.copyapikeybutton').on('click', oncopyapikeybuttonclick);
+        startCopyApiKeyButtonTimer()
```
Comment (dtivel) [inline src/NuGetGallery/Views/Users/ApiKeys.cshtml]: ;

**Pull request merged**